docusaurus.config.js

```diff
@@ -197,6 +202,14 @@ module.exports = {
         },
       ],
     },
+    algolia: {
+      apiKey: "4a2fa646f476d7756a7cdc599b625bec",
+      indexName: "temporal",
+      externalUrlRegex: "temporal\\.io",
+      // contextualSearch: true, // Optional, If you have different version of docs etc (v1 and v2) doesn't display dup results
```

Comment:
```suggestion
      // contextualSearch: true, // Optional; if you have different version of docs etc (v1 and v2), doesn't display dup results
```
What does "etc" mean in this context?